README.rst

```diff
@@ -157,16 +157,16 @@ Using this SDK you can create steps, chain them together to create a workflow,
 create that workflow in AWS Step Functions, and execute the workflow in the
 AWS cloud.
 
-.. image:: https://github.com/aws/aws-step-functions-data-science-sdk-python/raw/master/doc/images/create.png
-  :scale: 50 %
-  :alt: Create a workflow in AWS Step Functions
+.. image:: https://raw.githubusercontent.com/aws/aws-step-functions-data-science-sdk-python/master/doc/images/create.png
+    :width: 400
```

Comment:
Maybe 600px instead? 400 looks a little narrow.
```suggestion
    :width: 600
```

Reply:
The usual recommended pixel range is 400 - 800 so I think 600 makes sense. Will make the change